src/test/java/apoc/custom/CypherProceduresTest.java

```diff
@@ -258,6 +258,22 @@ public void shouldRemoveTheCustomFunction() throws Exception {
         TestUtil.count(db, "return custom.answer()");
     }
 
+    @Test
+    public void shouldOverwriteAndRemoveCustomProcedure() throws Exception {
+        db.executeTransactionally("call apoc.custom.asProcedure('answer','RETURN 42')");
+        db.executeTransactionally("call apoc.custom.asProcedure('answer','RETURN 42')");
+        assertEquals("Expecting one procedure listed", 1, TestUtil.count(db, "call apoc.custom.list()"));
```

Comment:
Can you also add a test for the system db that checks if there is more than 1 node?

Reply:
You mean like a separate test (what exatcly I would be testing? Unit test for `Util.mergeNode?`) or just a new assertions in these tests?

Reply:
No I mean a new assertion for this test that checks into the system db if there is just one node for the procedure/function as we removed the `Iterators.single` now would be better have a direct check into the system db

Reply:
`assertEquals("Expecting one procedure listed", 1, TestUtil.count(db, "call apoc.custom.list()"));` already does that, just indirectly.
And to do the check I would be more or less copying the implementation of `CypherProceduresHandler.readSignatures`, I'm not that comfortable querying db in code (no cypher) to write a bespoke query for the test.

Reply:
ok so let's leave it as is